src/category_theory/groupoid/basic.lean

```diff
@@ -0,0 +1,44 @@
+/-
+Copyright (c) 2022 Rémi Bottinelli. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Rémi Bottinelli
+-/
+import category_theory.groupoid
+import category_theory.is_connected
+
+/-!
+This file defines a few basic properties of groupoids.
+-/
+
+namespace category_theory
+
+namespace groupoid
+
+variables (C : Type*) [groupoid C]
+
+section thin
+
+/-- A groupoid is graph-like if it has no parallel arrows. -/
+def is_thin := ∀ (c d : C), subsingleton (c ⟶ d)
```

Comment:
I think this should be moved to `category/thin` and be defined for all categories (or all `quiver`).

Reply:
Ah, then probably adding that to `quivers/basic.lean` is enough?